### ISSUE TITLE
Dynarec CPU crashes Shield TV on new 5.1

### ISSUE THREAD

**Issue by fivefeet8** [devmiyax/yabause#336]
Every game will crash back to the emulator GUI when the CPU is set to Dynarec mode.  Games load fine with Interpreter CPU though.  Interpreter/dynarec modes works on my android Phone just fine.  It must be something with the latest update to the Shield.

**Comment by devmiyax** [comment]
What version do you use?

**Comment by fivefeet8** [comment]
I'm on the current 4.3 beta, but it also crashed on the stable 4.2.

**Comment by devmiyax** [comment]
I can't reproduce your problem. Please let me know more details. What game do you play? Didn't it happen on sheild 5.0?

**Comment by fivefeet8** [comment]
The Dynarec worked fine on Shield TV 5.0.x, but when I updated to 5.1 it started to crash any game.

**Comment by fivefeet8** [comment]
I've sent a log report to google play.  Hopefully that helps you track down the issue.  I'm still not able to run the Dynarec CPU core without the game crashing back to the uoYabause GUI.  Can anyone else with a Shield TV check if they have the same issue with Update Shield Update 5.2 and uoYabause?

**Comment by devmiyax** [comment]
```
*** *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***
Build fingerprint: 'NVIDIA/foster_e/foster:7.0/NRD90M/1915764_848.4973:user/release-keys'
Revision: '0'
ABI: 'arm'
pid: 13325, tid: 13348, name: d:emulationmain  >>> org.uoyabause.android:emulationmain <<<
signal 11 (SIGSEGV), code 1 (SEGV_MAPERR), fault addr 0x0
    r0 00000000  r1 60206e40  r2 60206e40  r3 00003800
    r4 55d14a7c  r5 00000000  r6 00000005  r7 00000000
    r8 00000000  r9 00000000  sl 00000000  fp 55629854
    ip 0007e4ba  sp 55629820  lr 55b897c8  pc 55b878d4  cpsr 800f0010

backtrace:
    #00 pc 0007d8d4  /data/app/org.uoyabause.android-2/lib/arm/libyabause_native.so (Cs2DeleteSectorData+184)
    #01 pc 0007f7c4  /data/app/org.uoyabause.android-2/lib/arm/libyabause_native.so (Cs2Exec+412)
    #02 pc 00112c28  /data/app/org.uoyabause.android-2/lib/arm/libyabause_native.so (cc_interrupt_master+320)
```

**Comment by fivefeet8** [comment]
Seems to have been fixed in the latest 0.4.3c.  Awesome!!!
